docs/developer/app-store/apps/taxes/avatax/overview.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Avatax
```

Comment:
```suggestion
title: Avalara AvaTax
```

Please update capitalization of the name in the whole document

Reply:
Good catch!